spacy/cli/debug_data.py

```diff
@@ -861,6 +898,20 @@ def _compile_gold(
                 data["n_nonproj"] += 1
             if nonproj.contains_cycle(aligned_heads):
                 data["n_cycles"] += 1
+        if "trainable_lemmatizer" in factory_names:
+            # NOTE: From EditTreeLemmatizer._labels_from_data
+            for token in eg.reference:
+                if token.lemma != 0:
+                    tree_id = trees.add(token.text, token.lemma_)
+                    tree_str = trees.tree_to_str(tree_id)
+                    data["lemmatizer_trees"].add(tree_str)
+    if "trainable_lemmatizer" in factory_names:
+        # After the edittree is built, let's try and capture all the
+        # string trees in that structure
+        all_trees = set()
+        for tree_id in range(len(trees)):
+            all_trees.add(trees.tree_to_str(tree_id))
+        data["lemmatizer_trees_full"] = all_trees
```

Comment:
`n=1120`

Reply:
What do these review comments refer to?

Reply:
I was noting the sizes of the various trees that were being generated while this was in draft, so it was easier to refer to them when reviewing. They can be closed.